src/dialog/dialog.component.ts

```diff
@@ -80,7 +83,8 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 	 */
 	constructor(
 		protected elementRef: ElementRef,
-		protected elementService: ElementService
+		protected elementService: ElementService,
+		protected animationFrameService: AnimationFrameService
```

Comment:
Since this is contained to the dialog base component can we make this optional? Otherwise it ends up as a breaking change ...

```suggestion
		@Optional() protected animationFrameService: AnimationFrameService = null
```